WorkManagerSample/app/src/main/java/com/example/background/ImageOperations.kt

```diff
@@ -28,12 +29,18 @@ import com.example.background.workers.*
 internal class ImageOperations private constructor(val continuation: WorkContinuation) {
 
     internal class Builder(private val mImageUri: Uri) {
+        private lateinit var mApplicationContext: Context
         private var mApplyWaterColor: Boolean = false
         private var mApplyGrayScale: Boolean = false
         private var mApplyBlur: Boolean = false
         private var mApplySave: Boolean = false
         private var mApplyUpload: Boolean = false
 
+        fun setApplicationContext(context: Context): Builder {
```

Comment:
If this is required to be set before build(), why isn't this part of the Builder's constructor?